app/src/main/java/org/liberty/android/fantastischmemo/utils/HintUtil.java

```diff
@@ -14,20 +16,34 @@
     public HintUtil() {
     }
 
-    public String generateLetterHint(int count, String word) {
+    public static String generateLetterHint(int count, String word, HashSet<Integer> used) {
```

Comment:
Another static used, why?

Reply:
Shouldn't have used static here but it is removed now.